lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedAccessControlConfig.java

```diff
@@ -56,4 +69,9 @@ public FileBasedAccessControlConfig setRefreshPeriod(Duration refreshPeriod)
         this.refreshPeriod = refreshPeriod;
         return this;
     }
+
+    public boolean isRest()
```

Comment:
Instead of checking if it a file or a REST url ? Can we have have some enum which could be configured so we could have dedicated set of properties for each type.

Reply:
refactored